File: java_backup/java.jimut/p_prog/finalpracexam/palindrome.java
package finalpracexam;


class palindrome
{
public static void main(int n)
{
int d,c,r=0;
c=n;
do
{
d=n%10;
r=r*10+d;
n=n/10;
}
while(n!=0);
if(r==c)
System.out.println("The no" +c+ "is Palindrome no");
else
System.out.println("The no" +c+ "is not Palindrome no");
}
}

File: 1st_and_2nd_Sem_c_backup/BIN/C Prog/practice c/palindrome.c

#include<stdio.h>
int main()
{
	int i,k=0,n;
	scanf("%d",&n);
	i=n;
	while(n!=0)
	{
		k=n/10;
		k=k+i%10;
		i=n%10;
	}
	if(n==k)
		printf("palindrome");
	else
		printf("not palindrome");
	return 0;
}
